### DISCUSSION TITLE
'flashcards' deck disappeared after upgrade to 1.6.0

### DISCUSSION THREAD

**Discussion by a42niem** [st3v3nmw/obsidian-spaced-repetition#242] [Q&A] [answered]
Hi all,
after upgrading from 1.5.7 to 1.6.0 when clicking on 'Review flashcards' the 'flashcards' deck is no longer listed. 
Playig around with the cards content did not help.
Then I deinstalled and manually installed version 1.5.8 and all is well again.
What might be wrong here? Thanks.

**Comment by st3v3nmw** [comment, marked as answer]
I think it's a caching issue, I'll do a fix in a few.

**Reply by st3v3nmw** [reply]
Are you on mobile or desktop? Brand(s)?

**Reply by a42niem** [reply]
I am on Debian Linux, XFCE4 and Obsidian is v0.12.12 if that helps

**Reply by a42niem** [reply]
I have now updated from 1.5.8 to 1.6.1 and it works as expected. Thank you very much for this plugin!
